digitaltwin_client/tests/dt_e2e/service/dt_e2e_test.ts

```diff
@@ -463,7 +463,10 @@ describe("DigitalTwin Properties E2E tests", function() {
         verifyPropertySet(done, testPropertyInterfaceName, "ProcessUpdatedProperty1", `ValueOfPropertyBeingSet_1`, 0)
     })
 
-    it("Verify all interfaces registered", function(done) {
-        veryAllInterfacesAreRegistered(done)
-    })
+    // Disable test requires updates from node service client sdk and the test code to support changes for Refresh.
```

Comment:
>// Disable test requires updates from node service client sdk and the test code to support changes for Refresh [](start = 4, length = 110)

Is their a work item somewhere tracking this work?

Reply:
yes, its tracked in devops TASK 6366438